EIPS/eip-5792.md

```diff
@@ -2,7 +2,7 @@
 eip: 5792
 title: Wallet Call API
 description: Adds JSON-RPC methods for sending multiple calls from the user's wallet, and checking their status
-author: Moody Salem (@moodysalem), Lukas Rosario (@lukasrosario), Wilson Cusack (@wilsoncusack), Dror Tirosh (@drortirosh), Jake Moxey (@jxom)
+author: Moody Salem (@moodysalem), Lukas Rosario (@lukasrosario), Wilson Cusack (@wilsoncusack), Dror Tirosh (@drortirosh), Jake Moxey (@jxom), Derek Rein (@arein)
```

Comment:
grazie

Reply:
Sorry, I'm confused. Did Derek write this PR or what's the idea here? Love Derek but there are a lot of people who have contributed to discussion at the same level who we haven't added.

Reply:
Definitely not opposed to adding more authors for contributions at author level! Just want to have a clear and fair criteria so we don't end up with like 20 authors.

Reply:
See commit breakdown in opening comment-- this was to close #7996 creditfully.  i can cherrypick out that commit if the other authors don't think 8387 (the PR we're in) merits a co-author cred.

Reply:
Ah I had not seen this PR, thanks! However it seems that, in that context, CAIP-25 is mentioned for discovering these RPCs? 

> ### Advertising support via [CAIP-25]

Whereas the debate here has been whether CAIP-25 should be used to discover capabilities? The strongest version of the argument being the argument that `wallet_getCapabilities` should not be present.

Reply:
oh I would not make that argument at all; I thought [i made the exact opposite argument a few lines below](https://github.com/ethereum/EIPs/pull/8387/files#diff-7a494500c37dc4b57613207d62874322198a4ced215f4a0457a0af908bb6364eR176). I think wallet RPCs should trump any three-party trust model and capability models should trump discovery models, but it's hard to thread that needle through today's business incentives and coordination landscape.

Reply:
Ok. I think I'm too smooth brain to follow :) 

But it seems like what is being discussed here is not to do with that prior PR? Or what's the part that was closely lifted?

Reply:
> Whereas the debate here has been whether CAIP-25 should be used to discover capabilities? The strongest version of the argument being the argument that wallet_getCapabilities should not be present.

I am NOT making the strongest version of the CAIP-25 maxi argument, even tho I have been working on CAIP-25 primarily for two years.  But I _AM_ making a strong argument for including non-normative mention of provider caching and provider-facilitated opt-in declaration of capabilities. subtle distinction, you're not smoothbrained for getting lost it's probably because I've been focused on this for 2 years that I can't explain it to a normal person.